src/Pub/Database/Eloquent/BroadcastsEvents.php

```diff
@@ -1,14 +1,12 @@
 <?php
 
-namespace PodPoint\SnsBroadcaster;
+namespace PodPoint\AwsPubSub\Pub\Database\Eloquent;
 
 use Illuminate\Database\Eloquent\BroadcastsEvents as EloquentBroadcastsEvents;
 
 trait BroadcastsEvents
```

Comment:
Is this file necessary? The newBroadcastableModelEvent function looks like a direct copy of the one it inherits from.

Reply:
It's not necessary anymore with Laravel 8.52.

The only thing different in this file was the use of our own `BroadcastableModelEventOccurred` instead of Laravel one.